lib/wibox/widget/graph.lua

```diff
@@ -416,8 +417,8 @@ function graph:draw_values(cr, width, height, drawn_values_num)
     end
 end
 
-function graph.fit(_graph)
-    return _graph._private.width, _graph._private.height
+function graph:fit(_, width, height)
+    return width, height
```

Comment:
This method should still check if `self._private.width` or `self._private.height` are set and honor them. Yes, I know, compatibility is annoying ;)